src/SFML/Window/Window.cpp

```diff
@@ -215,7 +222,7 @@ void Window::display()
     if (m_frameTimeLimit != Time::Zero)
     {
         sleep(m_frameTimeLimit - m_clock.getElapsedTime());
-        m_clock.restart();
+        m_frameTime = m_clock.restart();
```

Comment:
This does not work when there is no framerate limit.

Reply:
Oops you're right. Let me fix that.

Reply:
Should be fixed now